src/cli/run_command.zig

```diff
@@ -1280,13 +1290,144 @@ pub const RunCommand = struct {
         Output.flush();
     }
 
+    pub const ExecResult = union(enum) {
```

Comment:
can you add `signal: u32` here, this will make this similar to `std.process.Child.Term`. stopped and unknown are not as important.

and then i recommend adding some function in here to exit with the execresult, which will call exitWide for the codes, but use `raiseIgnoringPanicHandler` for signals. (posix only)

Reply:
Done